x-pack/plugins/maps/public/classes/sources/es_geo_grid_source/es_geo_grid_source.js

```diff
@@ -229,27 +265,21 @@ export class ESGeoGridSource extends AbstractESAggSource {
     return features;
   }
 
-  // Do not use composite aggregation when there are no terms sub-aggregations
-  // see https://github.com/elastic/kibana/pull/57875#issuecomment-590515482 for explanation on using separate code paths
-  async _nonCompositeAggRequest({
+  _addNonCompositeAggregationsToSearchSource(
```

Comment:
Can you rename `_addNonCompositeAggregationsToSearchSource` to `_addNonCompositeAggsToSearchSource`? Other function names in file abbreviate "aggregation" to "agg" and naming consistency should be maintained.

Reply:
fixed in https://github.com/elastic/kibana/pull/78201